.github/workflows/linux.yml

```diff
@@ -807,7 +807,7 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-22.04
+    runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-8-cores' || 'ubuntu-20.04'}}
     env:
```

Comment:
please, update after merging this PR https://github.com/openvinotoolkit/openvino/pull/19702

Reply:
Done